src/pages/Admin/Charity/Withdraws/Withdrawer/WithdrawForm/Network/Network.tsx

```diff
@@ -0,0 +1,88 @@
+import { Listbox } from "@headlessui/react";
+import { useEffect } from "react";
+import { useFormContext } from "react-hook-form";
+import { WithdrawValues } from "../types";
+import { useChainsQuery } from "services/apes";
+import Icon, { DrawerIcon } from "components/Icon";
+import QueryLoader from "components/QueryLoader";
+
+export default function Network() {
+  const queryState = useChainsQuery({});
+  const { register, watch, trigger } = useFormContext<WithdrawValues>();
+  const network = watch("network");
+
+  useEffect(() => {
+    (async () => {
+      //validate address on network change
+      await trigger("beneficiary");
+    })();
+  }, [network, trigger]);
+
+  return (
+    <div className="grid gap-3">
+      <span className="font-work font-bold">Select network</span>
+      <QueryLoader
+        queryState={queryState}
+        messages={{
+          loading: "Loading available networks...",
+          empty: "No available networks",
+          error: "Error loading networks",
+        }}
+      >
+        {(chains) => (
+          <Listbox {...register("network")} as="div" className="relative">
+            <Listbox.Button className="grid grid-cols-[1fr_auto] items-center text-left gap-2 w-full md:w-52 p-4 border border-prim rounded">
+              {({ open, value }) => (
+                <>
+                  <span className="truncate max-md:text-sm">
+                    {
+                      chains.find((chain) => chain.chain_id === value)
```

Comment:
network should be pre-populated per design
<img width="553" alt="image" src="https://user-images.githubusercontent.com/89639563/223012199-fd168898-25ba-467a-bdef-e6cbb8e719cd.png">